ci-values.yaml

```diff
@@ -24,6 +24,15 @@ rules:
     defaultAction: "allow"
     admClientmMde: "service"
     service: "neuvector"
+  response:
+    policies:
+      - '{"insert":{"after":0,"rules":[{"group":null,"conditions":[{"type":"name","value":"Admission.Control.Denied"}],"actions":["webhook"],"event":"admission-control","disable":false}]}}'
```

Comment:
is this safe to run over and over again? IIRC we might not be worrying about . that